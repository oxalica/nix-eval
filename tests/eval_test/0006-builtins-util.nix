{
  typeOf = with builtins; [
    (typeOf true)
    (typeOf 1)
    (typeOf 2.0)
    (typeOf { a = throw "set"; })
    (typeOf [ (throw "list") ])
    (typeOf [ (throw "list") ])
    (typeOf [ (x: throw "lam") ])
    (typeOf typeOf)
    (typeOf builtins.builtins)
  ];
}
