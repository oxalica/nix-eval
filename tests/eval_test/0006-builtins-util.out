{
  typeOf = [
    "bool"
    "int"
    "float"
    "set"
    "list"
    "list"
    "list"
    "lambda"
    "set"
  ];
}